source/includes/extracts-watch-option.yaml

```diff
@@ -1,14 +1,14 @@
 ref: watch-option-batchSize
 content: |
-  Specifies the batch size for the cursor, which will apply to both the initial
-  ``aggregate`` command and any subsequent ``getMore`` commands. This determines
-  the maximum number of change events to return in each response from the
+  The maximum number of documents within each batch returned in a query result, which applies 
```

Comment:
Consider replacing "query result" with "change stream" to make this more contextual:

> The maximum number of documents within each batch returned in a change stream

This pairs with my suggestion to remove a sentence below. Alternatively, you can try to work "change events" into this sentence, but I think having both "change events" and "change stream" is too repetitive.